trillian_map_api.proto

```diff
@@ -44,6 +51,10 @@ message MapLeaf {
 
 message MapLeafInclusion {
   MapLeaf leaf = 1;
+  // inclusion holds the inclusion proof for this leaf in the map root. It
+  // holds one entry for each level of the tree; combining each of these in
+  // turn with the leaf's hash (according to the tree's hash strategy)
+  // reproduces the root hash.
```

Comment:
Worth adding a bit about `nil`s in the inclusion proof in the "undefined" areas?

Reply:
Yeah, I meant to do that.